nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/PutMongoRecord.java

```diff
@@ -266,10 +265,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             error = true;
         } finally {
             if (!error) {
-                String url = clientService != null
```

Comment:
Is this always not null now?

Reply:
It should be because in 2.0 we're going client service only for configuring connections.